Update dependencies to match pyproject.toml

theavey opened the pull request:
update to match current version at https://github.com/gerrymanoim/exchange_calendars/blob/master/pyproject.toml

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->
kinda fixes #34 (would really need to go update the already deployed versions' metadata to actually fix it...)

<!--
Please add any other relevant info below:
-->

theavey committed:
Update dependencies to match pyproject.toml

update to match current version at https://github.com/gerrymanoim/exchange_calendars/blob/master/pyproject.toml

theavey commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ No valid build backend found for Python recipe for package `exchange-calendars` using `pip`. Python recipes using `pip` need to explicitly specify a build backend in the `host` section. If your recipe has built with only `pip` in the `host` section in the past, you likely should add `setuptools` to the `host` section of your recipe.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/12797144014](https://github.com/conda-forge/conda-forge-webservices/actions/runs/12797144014). Examine the logs at this URL for more detail.</sub>

Commit:
MNT: Re-rendered with conda-build 24.11.2, conda-smithy 3.45.2, and conda-forge-pinning 2025.01.15.19.32.56

theavey committed:
add build backend

was probably implicitly required with setuptools_scm, but better to be explicit

And very likely setuptools_scm not actually needed here when building from an sdist, not any scm

theavey commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/12797191693](https://github.com/conda-forge/conda-forge-webservices/actions/runs/12797191693). Examine the logs at this URL for more detail.</sub>
